packages/crayons-core/src/components/country-phone/country-phone.stories.mdx

```diff
@@ -0,0 +1,115 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+
+# CountryPhone
+
+<Meta
+  title='CountryPhone'
+  component='fw-country-phone'
+/>
+
+## Props 
+
+<Props of="fw-country-phone"/>
+
+## Usage
+
+### Defaults
+<Preview>
```

Comment:
Kindly add documentation for consuming as react component. Refer to other components for examples.